target/linux/sunxi/image/cortexa7.mk

```diff
@@ -108,6 +108,15 @@ define Device/linksprite_pcduino3
 endef
 TARGET_DEVICES += linksprite_pcduino3
 
+define Device/sun7i-a20-pcduino3-nano
+  DEVICE_TITLE:=LinkSprite pcDuino3 Nano
+  DEVICE_PACKAGES:=kmod-rtc-sunxi kmod-ata-sunxi
+  SUPPORTED_DEVICES:=linksprite,pcduino3-nano
+  SUNXI_DTS:=sun7i-a20-pcduino3-nano
+endef
```

Comment:
Since seems to be based on an outdated version. Please update it to match the syntax used in this file now.

Reply:
yes, this is what i try to porting openwrt 19.07, the patch is outdated.